torchvision/prototype/features/_feature.py

```diff
@@ -3,16 +3,14 @@
 from types import ModuleType
 from typing import Any, Callable, cast, List, Mapping, Optional, Sequence, Tuple, Type, TypeVar, Union
 
+import PIL.Image
 import torch
 from torch._C import _TensorBase, DisableTorchFunction
 from torchvision.transforms import InterpolationMode
 
 F = TypeVar("F", bound="_Feature")
-
-
-# Due to torch.jit.script limitation we keep DType as torch.Tensor
-# instead of Union[torch.Tensor, PIL.Image.Image, features._Feature]
-DType = torch.Tensor
+FillType = Union[int, float, Sequence[int], Sequence[float], None]
+FillTypeJIT = Union[int, float, List[float], None]
```

Comment:
Could move to another file if we don't want it here.

Reply:
I think it is ok there. I would prefer to have everything in a `_typing.py` file, but as you explained this will give us circular imports due to the method signatures. Thus, let's take the way of the least resistance.